server/src/main/java/org/apache/gravitino/server/web/rest/TableOperations.java

```diff
@@ -140,11 +167,23 @@ public Response createTable(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "load-table." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "load-table", absolute = true)
+  @AuthorizationExpression(
+      expression =
+          "( (METALAKE::USE_CATALOG || CATALOG::USE_CATALOG) && "
+              + "(METALAKE::USE_SCHEMA || CATALOG::USE_SCHEMA ||SCHEMA::USE_SCHEMA) &&"
+              + " (METALAKE::SELECT_TABLE || CATALOG::SELECT_TABLE || "
+              + "SCHEMA::SELECT_TABLE || TABLE::SELECT_TABLE || "
```

Comment:
Please use any expression like ANY(SELECT_TABLE,METALAKE,CATALOG,SCHEMA,TABLE)

Please refer to https://github.com/apache/gravitino/pull/7577/files